Update 14.biological-interpretation.md (#47)
jessegmeyerlab/proteomics-tutorial: content/14.biological-interpretation.md

## Biological Interpretation

As described in previous sections, quantitative proteomics experiments reveal protein abundance changes. 
After statistical data analysis, the typical output is a list of differentially expressed proteins that can contain hundreds of hits. 
The next step is how to interpret the results in biological terms. 
Often, it is tempting to focus on proteins with the most extreme fold changes. 
In this case, the assumption is that the more significant the fold change (in either direction, up- or down-regulation), the higher the impact of those proteins on cellular behavior. 
Alas, this assumption is not always valid. 

> Another approach is to start googling for every protein in the list to uncover groups that function together, maybe in protein complexes or signaling pathways. 

Reviewer: ```suggestion
Another approach is to start manually lookup every protein in the list to uncover groups that function together, maybe in protein complexes or signaling pathways. 
```

> This strategy is time-consuming and cumbersome, and it is almost impossible to capture all reported interactions from the wealth of the literature. 

Reviewer: ```suggestion
This strategy is time-consuming and cumbersome, and it is almost impossible to capture all reported interactions from the wealth of the literature. 
Therefore, automated computational methods to help with biological interpretation are needed. 
In this section, we overview some of these methods. 
```

### Constructing a protein network

A network is a representation of the relations between objects. 

> Nodes are the entities of the network (users of a social platform, train stations, proteins), while edges are the connections between them (friendship, routes, and protein interactions, respectively). 

Reviewer: ```suggestion
Nodes are the entities of the network (e.g., users of a social platform, train stations, proteins), while edges are the connections between them (e.g., friendship, routes, and protein interactions, respectively). 
```

In the case of protein-protein interactions, the evidence for the functional associations between proteins can be obtained experimentally. 
For example, co-immunoprecipitation, crosslinking, and proximity labeling can be used to reveal physical interactions [@PMID:34046695]. 
The data is presented in a table with nodes and edges (e.g., "protein A interacts with protein B") from which the network can be constructed. 
A considerable wealth of protein-protein association data is stored in free databases like IntAct, which contain interactions derived from literature curation or direct user submissions [@PMID:34761267]. 
Protein interactions can also be predicted by classifiers that consider many features, like orthology and co-localization, to produce a posterior odds ratio of interaction [@PMID:21572181; @PMID:33237311]. 
Finally, large repositories like STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) collect and integrate protein-protein interaction data from several databases [@PMID:33237311]. 

> It also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest. 

Reviewer: ```suggestion
STRING also provides a web-based interface to survey the data, and users only have to feed a search box with the identifiers of the protein(s) of interest. 
```

STRING will retrieve the network and show the evidence supporting each interaction. 
Importantly, these databases do not indicate the direction of the interaction, so they produce undirected networks. 

> If edges have directions (e.g., A influences B and not vice versa), then the network is directed. Signaling pathways are examples of directed graphs. 

Reviewer: ```suggestion
If edges have directions (e.g., A influences B and not vice versa), then the network is directed. 
Signaling pathways are examples of directed graphs. 
```

### Network analysis

Network analysis is a group of techniques that explore and investigate the network, yielding valuable knowledge about its structure and unveiling key players regulating the flow of information. 
One of the first steps in network analysis relates to centrality measurements. 
Centralities are indicators of the relative importance of a node corresponding to its position in the network, and each centrality measure provides new insights to interpret the data in new ways [@PMID:27616995; @PMID:30064421].

#### Degree centrality
  
The degree of a node measures the number of edges incident to that node. 
Nodes with a high degree interact with many other nodes, called first neighbors. 
In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors. 
Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network [@PMID:16751849]. 

> #### Closeness centrality

Reviewer: can you add a citation to the text below about closeness centrality where people can look for more information?

Author: Closeness centrality is not as heavily used as betweenness in protein networks. Still, I could add a nice citation showing that essential genes in yeast have a higher closeness than non-essential ones. I also added a link to a great resource at the European Bioinformatics Institute, but I don´t know if citing web pages is appropriate for this manuscript.

The route from one node to another is a path, and the shortest path is the one connecting them in the least amount of steps. 
Closeness centrality is the inverse of the average length of a node's shortest paths to all other nodes in the network. 
Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task.

#### Betweenness centrality

This centrality index is related to the amount of shortest paths transversing a node. 
Nodes with a high betweenness centrality usually bridge different parts of the network and strongly influence the flow of information, as they lie in communication paths. 
These connector hubs (or bottlenecks) are also interesting for follow–up experiments because their removal can disconnect different regions of the network [@PMID:17447836].

Centrality measurements add new layers of information and allow for ranking differentially expressed proteins apart from their fold-change in abundance. 

> Figure X depicts a simple network consisting of proteins A to L, with A having the highest fold change (10) and L the lowest (2). 

Reviewer: This figure is awesome thank you

In this regard, network clustering is useful for tentatively assigning the function of an uncharacterized protein. 
If the protein appears in a cluster, its function should be closely related to the cluster members, a principle known as "guilty by association." [@PMID:26215734]

### Network visualization

A critical step in network analysis is to display the data in a structured and uncluttered graph. 
Networks can rapidly become a hairball unamenable to interpretation. 
Software platforms like Cytoscape can be used to visualize networks orderly by applying layout algorithms and format styles [@PMID:14597658]. 
Since many of these platforms are open source, community-designed plugins enhance their capabilities. 
In Cytoscape, the stringApp adds a search bar to query the STRING database with accession numbers or protein names [@PMID:36512705]. 
The network is directly retrieved into Cytoscape, where its built-in network analyzer can be used to calculate centralities. 
Moreover, user-defined information, like fold-change values, can be integrated and mapped into the network.

### Functional enrichment analysis

> Gene and gene products have shared attributes regarding their cellular localization, molecular function, or biological process in which they take part, among others.

Reviewer: ```suggestion
Genes and gene products have shared attributes regarding their cellular localization, molecular function, or biological process in which they take part.
```

For example, citrate synthase and malate dehydrogenase are _mitochondrial_ enzymes that show _RNA binding_ activity and participate in the _Krebs cycle_. 
The terms in italics represent a fraction of the body of knowledge about these proteins. 
Similar information can be compiled and assigned to every gene or gene product, forming its gene ontology (GO): a set of concepts that describe the entity. 

> Functional enrichment analysis aims to reveal if genes or gene products grouped under a GO term (a.k.a., a "gene set") are over- or under-represented in a more extensive set. 

Reviewer: ```suggestion
Functional enrichment analysis assesses whether genes or gene products grouped under a GO term (a.k.a., a "gene set") are overrepresented in the set of proteins of interest compared to their normal frequency of occurrence in some background dataset. 
This is commonly achieved using Fisher's exact test [@DOI:10.2307/2340521].
```

For example, proteomics analysis of human cardiac 3D microtissue exposed to anthracyclines (drugs used in cancer chemotherapy) unearthed several proteins with altered levels [@PMID:33060801]. 
Many of these were specifically grouped under GO terms related to mitochondrial dysfunction, indicating the detrimental effects of these drugs on the organelle. 
GO terms [@PMID:10802651] or descriptors from other annotation libraries (like KEGG [@PMID:10592173] or REACTOME [@PMID:34788843]) can be retrieved from STRING when constructing a network or from other freely available compendiums. 

> Then, it is a matter of conducting a functional enrichment test to analyze which terms are over- or under-represented.

Reviewer: Over-representation analysis is very common but I'm not aware of under-representation analysis, do you have an example of that? If not can we change to just over-representation?

Author: Pardon my wording. For under-representation, I meant depletion. For clarity's sake, it is best to stick with over-representation for an introductory level.

> #### Over-representation analysis

Reviewer: this section seems redundant with the previous section - can you combine the two?

Author: I kept the three sub-sections as this better reflects the three most widely used types of functional enrichment methods (ORA, FCS, and PT). Instead, I tried to reduce redundancy by rewriting part of the introductory paragraph.

> In a proteomics analysis, thousands of proteins are identified and quantified. 

Reviewer: ```suggestion
In a modern proteomics analysis, usually thousands of proteins are identified and quantified. 
```

|-----------------------------|:-----------------------------------:|:-------------------------------------:|:---------:|
| **Proteins in TCA set**     |                  15                 |                   25                  |     40    |
| **Proteins not in TCA set** |                 185                 |                  1775                 |    1960   |
| **Total**                   |                 200                 |                  1800                 |    2000   |

	                    
A Fisher's exact test, or another suitable statistical test, is conducted to ascertain if proteins with altered levels are enriched in members of the TCA cycle (in this case, they are; p < 0.00001). 
This process is then repeated for every set as desired. 
Since multiple comparisons are tested, p values must be adjusted by a false discovery rate.

#### Functional class scoring
  
The caveat of over-representation methods is that they rely on a list of differentially expressed genes or proteins with altered abundance, selected due to arbitrarily chosen cut-off values. 

> So, a significant hit with a fold-change of 1.99 would not be included in the analysis. 

Reviewer: ```suggestion
For example, if we set a fold change cutoff of 2, a protein with a fold-change of 1.99 would not be included in the analysis. 
```

> ## Biological Interpretation

Reviewer: ```suggestion
## Biological Interpretation  {.page_break_before}
```

One of the first steps in network analysis relates to centrality measurements. 
Centralities are indicators of the relative importance of a node corresponding to its position in the network, and each centrality measure provides new insights to interpret the data in new ways [@PMID:27616995; @PMID:30064421].

#### Degree centrality
  
The degree of a node measures the number of edges incident to that node. 
Nodes with a high degree interact with many other nodes, called first neighbors. 
In particular, the node degree distribution in protein networks is highly skewed, with most nodes having a low degree and a few having high degrees, known as hubs.
Hubs are usually regulatory proteins, being notable examples oncogenes and transcription factors. 
Moreover, hubs are attractive targets for directed interventions, as their alteration has a profound effect on the stability of the network [@PMID:16751849]. 

#### Closeness centrality

The route from one node to another is a path, and the shortest path is the one connecting them in the least amount of steps. 
Closeness centrality is the inverse of the average length of a node's shortest paths to all other nodes in the network. 

> Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task [@PMID:15616139; @https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/].

Reviewer: ```suggestion
Nodes with a high closeness score have the shortest distances to all the others, so closeness centrality calculations detect nodes that can spread information very efficiently, as they are in a better position in the network for this task [@PMID:15616139; @URL:https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/building-and-analysing-ppins/topological-ppin-analysis/closeness-centrality/].
```
